tasks/extract_filepaths_for_granule/schemas/config.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "extract_filepaths_for_granule Lambda Config",
+    "description": "The config for the extract_filepaths_for_granule Lambda function.",
+    "type": "object",
+    "properties": {
+      "buckets": {
+        "type": "object",
+        "description": "AWS S3 bucket mapping used for Cumulus and ORCA configuration.",
+        "patternProperties": {
+          "\\S*": {
+            "description": "bucket configuration for the key'd bucket",
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "name of the S3 bucket"
+              },
+              "type": {
+                "type": "string",
+                "description": "the type of bucket - i.e. internal, public, private, protected"
+              }
+            }
+          }
+        }
+      },
+      "file-buckets": {
```

Comment:
Is this not required?

Reply:
yes, buckets and file-buckets should be required entities otherwise it will throw an error

Reply:
after adding this to the `config.json`,
```
    "required": [
      "buckets",
      "file-buckets"
    ],
```
I am getting unit test error in `test_handler_happy_path()` test. I tried manually putting values in config but still no luck. 

Error-> `jsonschema.exceptions.ValidationError: config schema: 'buckets' is a required property`

Reply:
added